### PR TITLE
Update certificates text

### PR DESCRIPTION
**Pull request by mel-am**
### What

Describe the change

### Why

Update certificates homepage text.

Link to JIRA card (if applicable):
[GW-1868](https://technologyprogramme.atlassian.net/browse/GW-1868)

[GW-1868]: https://technologyprogramme.atlassian.net/browse/GW-1868?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/views/certificates/index.html.erb [hunk 1]
```diff
@@ -5,7 +5,7 @@
       <h1 class="govuk-heading-l">Certificates</h1>
     </div>
       <div class="govuk-inset-text govuk-!-margin-top-0 govuk-!-margin-bottom-9">
-        You need to upload at least one Certificate to enable GovWifi Certificate Based Authentication. This will allow devices you issue (with a corresponding client certificate) to seamlessly authenticate with GovWifi.
+        You need to upload at least one Certificate to enable GovWifi Certificate Based Authentication. This will allow devices you issue (with a corresponding client certificate) to seamlessly authenticate with GovWifi. Certificate must be in .PEM format'
```
Comment (leonaAtkins) [inline app/views/certificates/index.html.erb:8]: Is there any associated unit test for this that might need the text updating?

Comment (mel-am) [reply app/views/certificates/index.html.erb:8]: There is already a format error message that appears if a .PEM file is not used.

Comment (mel-am) [reply app/views/certificates/index.html.erb:8]: it "reports an error" do
      file = Tempfile.new("bad_cert")
      file.write("this is not a certificate")
      file.close

      fill_in "Name", with: "MyCert1"
      attach_file("File", Rails.root.join(file.path))
      click_button "Upload Certificate"

      file.unlink

      expect(page).to have_content "Certificates must be in a .PEM format"
    end

Comment (leonaAtkins) [reply app/views/certificates/index.html.erb:8]: Cool, I was just checking if any existing test needed updating, but if all tests are passing, then it's all good.

**Pull request merged**